server/pages/index.js

```diff
@@ -342,13 +342,77 @@ function setUpLoggedInRoute( req, res, next ) {
 	}
 }
 
-function setUpRoute( req, res, next ) {
-	if ( req.cookies.wordpress_logged_in ) {
-		// the user is probably logged in
-		setUpLoggedInRoute( req, res, next );
-	} else {
-		setUpLoggedOutRoute( req, res, next );
+/**
+ * Sets up a Content Security Policy header
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+ * @param {Object} req Express request object
+ * @param {Object} res Express response object
+ * @param {Function} next a callback to call when done
+ */
+function setUpCSP( req, res, next ) {
```

Comment:
nit pick you can ignore, but it `setUp` usually split at the `U`? I'm more accustomed to seeing `setupCSP` or even `setupCsp` but in any case `setup` as a single word.

Reply:
I used the same case `setUpLoggedInRoute` and `setUpLoggedOutRoute` were already using

Reply:
you prompted me to look it up :smile:

"setup" is a noun generally
"set up" is the verb form

for a reasonable explanation
https://www.visualthesaurus.com/cm/wc/is-it-set-up-set-up-or-setup/

in the several articles I read which seemed to harmonize quite well it
appears like your use of `setUpCSP()` is more appropriate for middleware
which _does_ things while it might take someone to the _`cspSetup`_ page,
for instance.

TIL